static/app/components/modals/demoEndModal.tsx

```diff
@@ -0,0 +1,195 @@
+          method: 'PUT',
+          data: {guide, status: 'restart'},
+        })
+      )
+    ).then(() => {
+      if (tour === 'issues') {
+        localStorage.setItem('issueGuide', '1');
+      }
+      if (tour === 'tabs') {
+        localStorage.setItem('sidebarGuide', '1');
+      }
+
+      if (closeModal) {
+        closeModal();
+      }
+
+      fetchGuides();
+
+      const redirectUrl = new URL(location, window.location.origin);
+      redirectUrl.searchParams.append('referrer', 'demo_task');
+      navigateTo(redirectUrl.toString(), router);
+    });
+  }
+
+  const handleMoreTours = () => {
+    if (closeModal) {
+      closeModal();
+    }
+    togglePanel(SidebarPanelKey.OnboardingWizard);
+  };
+
+  return (
+    <EndModal>
+      <CloseButton
+        size="zero"
+        onClick={() => {
+          if (closeModal) {
+            closeModal();
+          }
+        }}
+        aria-label={t('Close')}
+        icon={<IconClose size="xs" />}
+      />
+      <Body>
+        <h2> {t('Tour Complete')} </h2>
+      </Body>
+      <ModalTask title={tct('[title]', {title: cardTitle})} />
+      <Body>
+        {tct(
+          'Thank you for [body]. Learn about other Sentry features by starting another tour.',
+          {body}
+        )}
+      </Body>
+      <ButtonContainer>
+        <a href={url} target="_blank" rel="noopener noreferrer">
+          <SignUpButton>
+            {tct('[prompt]', {prompt: sandboxData?.cta?.title || 'Sign up for Sentry'})}
+          </SignUpButton>
+        </a>
+        <ButtonBar>
+          <ModalButton onClick={handleMoreTours}> {t('More Tours')} </ModalButton>
+          <ModalButton onClick={handleRestart}>{t('Restart Tour')}</ModalButton>
+        </ButtonBar>
+      </ButtonContainer>
+    </EndModal>
+  );
+}
+
+export const modalCss = css`
+  width: 100%;
+  max-width: 500px;
+  [role='document'] {
+    position: relative;
+    padding: 50px 60px;
+  }
+`;
+
+const EndModal = styled('div')`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  align-items: center;
+`;
+
+const Body = styled('div')`
+  p {
+    font-size: 16px;
+    text-align: center;
+    margin: 0;
+  }
+  h2 {
+    font-size: 2em;
+    margin: 0;
+  }
+`;
+
+const CloseButton = styled(Button)`
+  position: absolute;
+  right: -15px;
+  top: -15px;
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  background-color: ${p => p.theme.background};
+  border: 1px solid ${p => p.theme.border};
+`;
+
+const ModalButton = styled(Button)`
+  background-color: ${p => p.theme.white};
+`;
+
+const SignUpButton = styled(Button)`
+  background-color: ${p => p.theme.purple300};
+  border: none;
+  color: ${p => p.theme.white};
+  width: 100%;
+`;
+
+const ButtonBar = styled('div')`
+  display: flex;
+  flex-direction: row;
+  gap: 5px;
+  justify-content: center;
+`;
+const ButtonContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`;
+
+export default DemoEndingModal;
```

Comment:
should we use named export ?

Reply:
is there a reason why this would be needed? i was going off of the other Modals which all seem to be default exports

Reply:
we are switching to named export but I think that is fine